src/polyswarm_api/__main__.py

```diff
@@ -43,6 +43,25 @@ def _is_valid_sha256(value):
         return False
     return is_hex(value)
 
+def _get_hash_type(value):
```

Comment:
this same function is in the `__init__.py`. It should only be in one place.

Reply:
Rather than just checking length, should we just use the existing `_is_valid_HASHTYPE` functions here?

Reply:
done